typescript/sdk/src/token/Token.ts

```diff
@@ -175,6 +196,7 @@ export class Token implements IToken {
   getHypAdapter(
     multiProvider: MultiProtocolProvider<{ mailbox?: Address }>,
     destination?: ChainName,
+    fillDeadline: number = Math.floor(Date.now() / 1000) + 60 * 60 * 24,
```

Comment:
`fillDeadline` is an intent-specific value that can only be accessed if it's passed via parameters at runtime.

It has a default value to 1 day (in seconds) if it's not defined in the request.